data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -221,10 +226,46 @@ def write_low_to_file():
             )
             logger.info("Completed writing usa-low")
 
+        def write_esri_shapefile():
```

Comment:
this is pretty gross. if you guys have a suggestion for how to streamline it, i am ALL EARS.